src/MOI_wrapper.jl

```diff
@@ -136,3 +136,14 @@ end
 function MOI.get(opt::Optimizer, ::MOI.NodeCount)
     return MOI.get(opt.oa_model, MOI.NodeCount())
 end
+
+"""
+    NumberOfCuts
+
+Number of cuts added to the outer approximation model.
+"""
+struct NumberOfCuts <: MOI.AbstractOptimizerAttribute end
```

Comment:
```suggestion
struct NumberOfCuts <: MOI.AbstractModelAttribute end
```

Reply:
is that a model attribute if it's set by optimizing? It's not tied to the model itself but the solving process